packages/@aws-cdk/aws-pipes-alpha/README.md

```diff
@@ -420,95 +397,44 @@ targets are supported:
 The target event can be transformed before it is forwarded to the target using
 the same input transformation as in the enrichment step.
 
-### Example target implementation
-
-> Currently no implementation exist for any of the supported targets. The following example shows how an implementation can look like. The actual implementation is not part of this package and will be in a separate one.
-
-```ts fixture=pipes-imports
-class SqsTarget implements pipes.ITarget {
-  targetArn: string;  
-  private inputTransformation: pipes.InputTransformation  | undefined;
-
-  constructor(private readonly queue: sqs.Queue, props: {inputTransformation?: pipes.InputTransformation} = {}) {
-    this.queue = queue;
-    this.targetArn = queue.queueArn;
-    this.inputTransformation = props?.inputTransformation
-  }
-
-  bind(_pipe: pipes.Pipe): pipes.TargetConfig {
-    return {
-      targetParameters: {
-        inputTemplate: this.inputTransformation?.bind(_pipe).inputTemplate,
-      },
-    };
-  }
+### Example target
 
-  grantPush(pipeRole: cdk.aws_iam.IRole): void {
-    this.queue.grantSendMessages(pipeRole);
-  }
-}
+```ts
+declare const targetQueue: sqs.Queue;
+const pipeTarget = new SqsTarget(targetQueue);
 ```
 
-A target implementation needs to provide the `targetArn`, `enrichmentParameters` and grant the pipe role invoke access to the enrichment.
-
 ## Log destination
 
 A pipe can produce log events that are forwarded to different log destinations. 
 You can configure multiple destinations, but all the destination share the same log level and log data.
 For details check the official [documentation](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-logs.html). 
 
 The log level and data that is included in the log events is configured on the pipe class itself.
-Whereas the actual destination is defined independent.
-
-### Example log destination implementation
+The actual destination is defined independently, and there are three options:
 
-> Currently no implementation exist for any of the supported enrichments. The following example shows how an implementation can look like. The actual implementation is not part of this package and will be in a separate one.
```

Comment:
It seems like the expected approach for this module is to have a new folder for the log destinations, likely akin to how we have `aws-pipes-sources-alpha` and `aws-pipes-targets-alpha`. We should probably follow this guideline, but I'll bring it up with the CDK team to confirm and get back to you, since it may add more confusion splitting one module into so many different parts.

Reply:
Yeah, I'd prefer to leave these in `aws-pipes` but open to feedback from the team.

Reply:
It seems like there is an argument to be made about having this be in the `aws-pipes-alpha` module, but there isn't really much extra cost having a separate `aws-pipes-log-destinations-alpha` folder, or something of that nature. One of the main reasons these are split the way they are is to prevent the risk of circular dependencies, and I'm not sure how big of a risk that could be in this case.

I think we _could_ leave things as is, but I would lean more towards making a separate module for log destinations, and in the future before graduating this module, see if we can refactor things and consolidate everything into one. Thoughts?

Reply:
I'd prefer to leave them here. Logging configuration is tied to a pipe, so I think it makes sense. But will of course default to the you and the team.

Reply:
Also, kinesis is now in dev preview which I think was the concern with circular dependency?

Reply:
Not sure how dev preview might be a circular dependency concern; if anything I think the risk with that is, when it's out of dev preview and into GA, the import will need to be updated, but that's expected. It more so applies to the module as a whole, but we can keep the log destination resources here, and investigate refactoring at a later time.